web/src/main/webapp/WEB-INF/pages/candidate_create.jsp

```diff
@@ -3,6 +3,7 @@
 <%@taglib uri="http://www.springframework.org/tags" prefix="spring" %>
```

Comment:
Be consistent, put prefix as first attribute, so the are aligned nicely below each other, and can be easily read.